Tests/KlaviyoCoreTests/ArchivalUtilsTests.swift

```diff
@@ -105,7 +106,6 @@ class ArchivalSystemTest: XCTestCase {
     let TEST_URL = filePathForData(apiKey: "foo", data: "people")
 
     override func setUpWithError() throws {
-        environment = KlaviyoEnvironment.production
```

Comment:
why remove this?

Reply:
I don't remember now but may be the test was passing and indeed using the production environment in `unarchiveFromFile` but might be good to explicitly specify it just in case some other test overrides the `environment` variable and renders this test as flaky. Will add it back in.